tests/docker/wp-dev-lib/Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG WP_PHP_VERSION=7.4
+FROM wordpress:php${WP_PHP_VERSION}-fpm
+
+# Allow custom versions of WP to be pulled in.
+ARG WP_SVN_URL=https://develop.svn.wordpress.org/trunk/
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Development tooling dependencies.
+RUN apt-get update \
+	&& apt-get install --yes --no-install-recommends \
+		bash less subversion default-mysql-server default-mysql-client libxml2-utils rsync git zip unzip \
+		nodejs npm curl \
+	&& npm install --global npm@latest \
```

Comment:
@paulschreiber This is a fix for that "Warning about old npm:" message. Here we bump it up.